tap_intercom/intercom.py

```diff
@@ -21,7 +22,6 @@ class Intercom:
     }
 
     def __init__(self, access_token, tap_stream_id):
-        self.SESSION = requests.Session()
```

Comment:
No no, you misunderstand. The session is vital to keep in the init. it is the `stap_stream_id` that should not be in the constructor.

Reply:
why it has to be in the init? so when the object is recycled, the session also end?

Reply:
It so that the session is onlt taking up memory after the instance is instantiated. If it at the package level, then whenever someone is importing the package, it will start the session even though the app might never use the class. It means we are wasting resources when we don't need to, basically.